### PR TITLE
[Feature] Specs for RVML

### PR DESCRIPTION
**Pull request by rvyuha**
Contains approved parts from rusty-parts as well as modifications to make it compatible with main branch
-Added new directory inside the qmd folder to store templates
-Organized raw and processed data inside the rvml folder
-Made minor modifications to rvml-parsing.R to allow for processing without the creation of en and fr directories

### CODE REVIEW

**Comment by rvyuha** [comment]
Parts qmd is found in rvml/content/en

**Review by yulric — CHANGES_REQUESTED** [review]
1. Can you point to the generated parts.qmd file? I don't see it in the PR
2. Can you add a comment to the gitignore to your line updates
3. The templates should probably be in their own folder in the root, to ensure they don't rendered to the website by mistake
4. Can we try to move all the rvml files to the root?
    1. The R folder can be moved to the root
    2. All spec files can go in a specs folder
    3. The data folder can be moved to the root
5. Can you make sure that all the files follow the naming convention outlines in the repo README?

**Review by yulric — APPROVED** [review]

**Review by yulric — CHANGES_REQUESTED** [review]
Don't think you did this, "Can you add a comment to the gitignore to your line updates"

**Review by yulric — APPROVED** [review]

**Review by yulric — APPROVED** [review]

**Comment by yulric** [comment]
@DougManuel Can I merge this into master?

File: parts-list-specs/updated-parts-list-spec.R [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
+[partLabel]([partID]): [partDesc] [partInstr; if == NA, print = FALSE]
+"Part Type:" [partType]
+"Domain ID:" [domainID]
+"Specimen ID:" [specimenID]
+"Compartment Set:" [compartSet]
```
Comment (DougManuel) [inline parts-list-specs/updated-parts-list-spec.R:6]: compSet?

Comment (yulric) [reply parts-list-specs/updated-parts-list-spec.R:6]: @rvyuha Can you update this line and then merge it into master?

**Review by DougManuel — APPROVED** [review]
Looks good. 

Note my comment for comparment set. The part name may not be correct for the most recent version of the parts table (that will be pushed today).

I have a few comments, but I'll hold them back for this PR.

**Review by DougManuel — COMMENTED** [review]
Do we need/want the .Rproj file?

**Comment by yulric** [comment]
> Do we need/want the .Rproj file?

I would keep only so the folder can be opened and edited in RStudio.

**Pull request merged**